params/config.go

```diff
@@ -120,6 +161,8 @@ const (
 type ChainConfig struct {
 	ChainID *big.Int `json:"chainId"` // chainId identifies the current chain and is used for replay protection
 
+	Incompatible1Block *big.Int `json:"incompatible1Block"` // incompatible1 switch block (nil = no fork, 0 = already incompatible1)
```

Comment:
This is a minor comment and a matter of taste, wouldn't it be better if the number is located at the end of the variable name for better readability? Like `IncompatibleBlock1`.

Reply:
@ehnuje Thanks for the comment, i didn't think about it.. However, i'm planning to change the name of Incompatible1 to IstanbulCompatible :). Are you okay with this name? It means a compatible with istanbul hardfork.

Reply:
@yoomee1313 If so, `IstanbulCompatible` "only includes" the changes of Istanbul hardfork or "also includes" the changes of Klaytn hardfork?

Reply:
@ehnuje If this only includes Istanbul hardfork, then i wouldn't have named it IstanbulCompatible. In other words, Klaytn's hard fork items will also be included.

Reply:
@yoomee1313 Thanks for the explanation. I understood. 😄